error[E0423]: expected function, found struct `Vec`
  --> $DIR/issue-40396.rs:20:38
   |
LL |     println!("{:?}", (0..13).collect<Vec<i32>()); //~ ERROR expected function, found struct `Vec`
   |                                      ^^^^^^^^ did you mean `Vec { /* fields */ }`?

error[E0615]: attempted to take value of method `collect` on type `std::ops::Range<{integer}>`
  --> $DIR/issue-40396.rs:20:30
   |
LL |     println!("{:?}", (0..13).collect<Vec<i32>()); //~ ERROR expected function, found struct `Vec`
   |                              ^^^^^^^
   |
   = help: maybe a `()` to call it is missing?

error: aborting due to 2 previous errors

Some errors occurred: E0423, E0615.
For more information about an error, try `rustc --explain E0423`.
